client/components/profilePage/userBookClubs.jsx

```diff
@@ -4,30 +4,48 @@ import { AppContext } from '../../context/context.jsx';
 import { AuthContext } from '../../context/authContext.jsx';
 import { Card, Col, Row, Tab, Tabs } from 'react-bootstrap';
 import './profileStyles.css';
+import axios from 'axios';
+import { Link } from 'react-router-dom';
 
-export default function UserBookClubs({user = {bookclubs: []}}) {
+export default function UserBookClubs({ user = { bookclubs: [] } }) {
+  const usrId = JSON.parse(localStorage.getItem('usrID')) || '';
```

Comment:
The way around not using localstorage is to conditionally run the axios call in the below use effect based on whether user is defined, and to have the use effect watch the user parameter